sdk/communication/Azure.Communication.CallingServer/README.md

```diff
@@ -35,6 +35,14 @@ var connectionString = "<connection_string>"; // Find your Communication Service
 CallingServerClient callingServerClient = new CallingServerClient(connectionString);
 ```
 
+```C# Snippet:Azure_Communication_CallingServer_Tests_Samples_CreateCallingServerClientWithToken
+string endpoint = "<endpoint_url>";
+string userAssignedClientId = "<your managed identity client Id>";
```

Comment:
In general, let's keep the format of the snippet as well as the place holder string names consistent with existing README: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/communication/Azure.Communication.Identity/README.md

Reply:
Done.